### ISSUE TITLE
Cause a ImportWarning when using lxml

### ISSUE THREAD

**Issue by tadashi-aikawa** [tadashi-aikawa/jumeaux#125]
```
ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
```

**Comment by tadashi-aikawa** [comment]
Watch https://bugs.launchpad.net/lxml/+bug/1732505

**Comment by tadashi-aikawa** [comment]
Fix lxml >= 4.2.6

**Comment by tadashi-aikawa** [comment]
Maybe Fix 0.65.2
